### ISSUE TITLE
Error while open select with helper-elements=""

### ISSUE THREAD

**Issue by silviotroia** [isteven/angular-multi-select#221]
Cannot read property 'getElementsByTagName' of undefined

it depend on  
var checkboxes = element.children().children().next().children()[0].getElementsByTagName('input');

to solve 
// var checkboxes = element.children().children().next().children()[0].getElementsByTagName('input');
// for ( var i = 0; i < checkboxes.length ; i++ )      { formElements.push( checkboxes[ i ] ); }

**Comment by tedyoung** [comment]
Looks like this is a duplicate of #206.

**Comment by forrestab** [comment]
Instead of having the empty helper-elements property, I ended up adding the following css.

```css
.multiSelect .checkboxLayer .helperContainer {
    display: none;
}
```

**Comment by isteven** [comment]
Fixed on v4.0.0.
